graylog2-server/src/main/java/org/graylog/events/search/MoreSearch.java

```diff
@@ -298,8 +319,21 @@ public void scrollQuery(String queryString, Set<String> streams, TimeRange timeR
         return streams;
     }
 
+    private String decorateQuery(Set<Parameter> queryParameters, TimeRange timeRange, String queryString) {
+        org.graylog.plugins.views.search.Search search = org.graylog.plugins.views.search.Search.builder()
```

Comment:
Can you please document why we have to use search job and query dummy objects here and that we probably want to refactor the ES query decorator do allow easier access to parameter substitution. Thanks!